internal/handler/hellohandler.go

```diff
@@ -0,0 +1,15 @@
+package handler
```

Comment:
Since the folder and package name is already "handler", we don't need "handler" in the filename, what do you think?
```
internal/handler/hello.go
```